lib/OptimizationMOI/Project.toml

```diff
@@ -19,7 +20,7 @@ Ipopt = "1"
 Ipopt_jll = "300.1400"
 Juniper = "0.9"
 MathOptInterface = "1"
-ModelingToolkit = "8.74"
+ModelingToolkit = "9"
```

Comment:
The downstream MOI one fails becuase you have removed 8.74 here but not in Optimization.jl's project.toml